src/server/protocol.ts

```diff
@@ -764,6 +766,28 @@ namespace ts.server.protocol {
         body?: string[];
     }
 
+    /**
+     * A request to get encoded Syntactic classifications for a span in the file
+     */
+    /** @internal */
+    export interface EncodedSyntacticClassificationsRequest extends FileRequest {
+        arguments: EncodedSyntacticClassificationsRequestArgs;
+    }
+
+    /**
+     * Arguments for EncodedSyntacticClassificationsRequest request.
+     */
+    export interface EncodedSyntacticClassificationsRequestArgs extends FileRequestArgs {

```

Comment:
This should be internal since command is internal

Reply:
Can I update `EncodedSemanticClassificationsRequestArgs` or would that be a breaking change at this point?

Reply:
I am not sure if any one uses it (other than us) but since that method is internal too, its ok to make it internal in my opinion